pkg/build/builder/sti.go

```diff
@@ -70,6 +70,7 @@ func (s *STIBuilder) Build() error {
 		return err
 	}
 	defer removeImage(s.dockerClient, tag)
+	glog.V(4).Infof("Starting STI build from %s BuildConfig ...", s.build.Name)
```

Comment:
Don't use "BuildConfig", use "build configuration.  Don't use "STI".  Use "Source" or "source to image" or "S2I".